content/zh-cn/docs/reference/kubernetes-api/other-resources/validating-admission-policy-binding-list-v1beta1.md

```diff
@@ -0,0 +1,28 @@
+---
+api_metadata:
+  apiVersion: "admissionregistration.k8s.io/v1beta1"
+  import: "k8s.io/api/admissionregistration/v1beta1"
+  kind: "ValidatingAdmissionPolicyBindingList"
+content_type: "api_reference"
+description: ""
+title: "ValidatingAdmissionPolicyBindingList v1beta1"
+weight: 1
+auto_generated: true
```

Comment:
Should we remove this line? It is manually copied currently

Reply:
yes, thanks for noticing